spec/requests/admin_requests_spec.rb

```diff
@@ -8,11 +8,31 @@
       get admin_dashboard_path
       expect(response).to be_successful
     end
+
     context "with rendered views" do
+      let!(:users_list) { create_list(:user, 25) }
+      let!(:user) { create(:user) }
+
       it "shows a logout button" do
         get admin_dashboard_path
         expect(response.body).to match(/log out/im)
       end
+
+      it "shows recently added users email" do
+        get admin_dashboard_path
+        expect(response.body).to match(/#{user.email}/im)
+      end
+
+      it "shows only 20 users within recently added users ssection" do
+        get admin_dashboard_path
+        expect(response.body).to match(/20 New users/im)
+      end
+
+      it "displays edit users path on recently added users ssection" do
+        get admin_dashboard_path
+        edit_user_path_regex = Regexp.new('admin/users/[0-9]+/edit\\?organization_id=admin')
+        expect(response.body).to match(edit_user_path_regex)
```

Comment:
I think it's OK to merge these 3 tests together (one setup / execution, 3 assertions). Having to create 75 users is pretty heavy for a test.